orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/Front50Service.groovy

```diff
@@ -159,6 +159,9 @@ interface Front50Service {
   @DELETE("/applications/{application}/deliveries/{id}")
   Response deleteDeliveryConfig(@Path("application") String application, @Path("id") String id)
 
```

Comment:
Why can't you use the DELETE above? Has the same arguments?

Reply:
Removed the new method.